src/core/events/controllers/event.controller.ts

```diff
@@ -0,0 +1,21 @@
+import { Controller, Get, Post, Body, Param } from '@nestjs/common';
+import { EventsService } from '../services/events.service';
+import { Event } from 'src/models/event.model';
+import { EventDto } from '../dtos/event.dto';
+
+@Controller('events')
+export class EventsController {
+  constructor(private readonly eventService: EventsService) {}
+  @Get()
+  async find(): Promise<Event[]> {
+    return this.eventService.find();
+  }
+  @Get(':group')
+  async getEvent(@Param('group') group: string): Promise<Event> {
+    return this.eventService.getEvent(group);
+  }
+  @Post()
```

Comment:
kolejnosć to od mniej inwazyjnego do bardziej, czyli get, get id, create i delete

Reply:
nazwa to starczy `create`, `find` i `get` dla tego z id